trie/verkle.go

```diff
@@ -118,6 +118,16 @@ func (trie *VerkleTrie) Commit(onleaf LeafCallback) (common.Hash, int, error) {
 	flush := make(chan verkle.VerkleNode)
 	go func() {
 		trie.root.(*verkle.InternalNode).Flush(func(n verkle.VerkleNode) {
+			if onleaf != nil {
+				if leaf, isLeaf := n.(*verkle.LeafNode); isLeaf {
+					for i := 0; i < verkle.NodeWidth; i++ {
+						if leaf.Value(i) != nil {
+							comm := n.ComputeCommitment().Bytes()
```

Comment:
Lots of redundant calls to `ComputeCommitment` here.  Maybe I am missing something?

Reply:
the commitment is cached, so it's not a problem. Unless I misunderstand your point?

Reply:
Oh whoops yeah you are right.